### DISCUSSION TITLE
The Game Receiving Abandon for Me Although I Never Launch?

### DISCUSSION THREAD

**Discussion by ghost** [ValveSoftware/Dota2-Gameplay#5934] [Q&A]
Very weird. Match ID 6914886001 recorded my account as abandoned with one match but I can swear on this that I never start up my Dota 2 application at all to play any game during that time. Can this be a bug of the game as recently I always can see players' disconnection, especially during the early or start of a game? Besides, I also saw/faced another graphic glitch triggered in a different game where a Tusk hero rolled but the "snowball" can still jam on a certain location (coordination) and allow the caster to roll it later although the hero moved another far location already.

Most clear is, when I attempted to play a game and check my matches, I was shocked to see there is an abandoned match recorded there as mentioned. I can say there is zero penalty given on that abandon at all. Funny as the game simply recorded an abandon to me but never provide any penalty at all.

**Comment by sellerdash09** [comment]
I have faced a similar issue[.](https://sellerdash.pk/) I was about to report it to help desk, as I can see it has not been solved here yet. Will share the solution.

**Reply by ghost** [reply]
> I have faced a similar issue[.](https://sellerdash.pk/) I was about to report it to help desk, as I can see it has not been solved here yet. Will share the solution. 

Seems I am not alone. I can believe many had faced this issue already especially an abandon record appears in the match histories. 

Recently (yesterday), happened to my party mate also where I saw with my eyes. Our match was loading and awaiting for a player from another side to be fully connected, but the system said that guy failed to connect. Then, the match jumped out and my party mate was the victim who received a penalty of 2 minutes abandoned. We were shocked also when the FIND MATCH button turned red. 

Big question is, why my party mate suddenly received the 2 minutes penalty for abandoned? Zzz...
